gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Fri May 01 11:14:42 PDT 2020
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.1-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

Comment:
we might also want to downgrade the jar for these to be in sync and not sure if the `gradlew` needs downgrading as well.